server/app/views/applicant/ProgramCardsSectionFragment.html

```diff
@@ -28,20 +28,26 @@
 
 <li
   th:fragment="card (card, sectionType)"
+  th:with="showImage=(${sectionType != 'MY_APPLICATIONS' and card.imageSourceUrl().isPresent() and not #strings.isEmpty(card.imageSourceUrl().get())})"
   class="usa-card tablet-lg:grid-col-6 widescreen:grid-col-4 cf-application-card"
+  th:classappend="${showImage} ? usa-card--header-first : '' "
 >
   <div class="usa-card__container">
-    <div class="usa-card__header">
-      <img
-        th:if="${sectionType != 'MY_APPLICATIONS' and card.imageSourceUrl().isPresent() and not #strings.isEmpty(card.imageSourceUrl().get())}"
-        th:src="${card.imageSourceUrl().get()}"
-        th:alt="${card.altText().orElse('')}"
-        class="cf-program-card-image"
-      />
-      <h2
-        class="usa-card__heading cf-application-card-title"
+    <div class="usa-card__header usa-prose">
+      <h3
```

Comment:
**praise:** Good catch that this should be an `h3` rather than an `h2`!

Reply:
Thanks!